compiler/qsc_rir/src/passes/reindex_qubits/tests.rs

```diff
@@ -236,7 +236,7 @@ fn qubit_reindexed_multiple_times_with_mz_inserts_multiple_cx() {
     let mut program = Program::new();
     program.num_qubits = 1;
     program.callables.insert(X, x_decl());
-    program.callables.insert(MZ, mz_decl());
+    program.callables.insert(MZ, m_decl());
```

Comment:
This symbol didn't get updated like the rest. Up to you if you want to do it now or not.

Reply:
Updated.